### PR TITLE
Dropdown and theme button fixing in Course and Index file.

### LINKED ISSUE #193: [Bug]:  Dropdown button is not working.
### Describe the bug

The hamburger icon is not working while I'm trying with "Toggle device Toolbar" and inspect it.

### To Reproduce

1. Open Course page.
2. _**Right-click**_ and **_inspect_** it
3. Click on the **_Toggle device toolbar_**.
4. Do minimize webpage till Hamburger Icon doesn't show.
4. Now try to Open the DropDown menu.
5. See the error.

### Expected Behavior

On click on the hamburger icon, The Dropdown menu should be open.

### Desktop (please complete the following information)

Device - HP
OS - Windows 10 Home
Browser:- Google Chrome
Version:- Version 94.0.4606.71 (Official Build) (64-bit) 

### Additional context

_No response_

### LINKED ISSUE #208: [Bug]: Issue with Toggle Button when Theme Changes
### Describe the bug

Change Theme Button has no fixed State 

### To Reproduce

- Change the theme
- Refresh the page
- State of button automatically changes but theme remains same

### Expected Behavior

It should check which theme is Currently in Local Storage and then then set the UI Accordingly

### Screenshot/ Video

image

image

### Desktop (please complete the following information)

Device [Laptop]
OS [Windows 10]
Browser [Chrome]

### Additional context

You can clearly see moon Icon is in both in dark and light mode
But it should appear Either in Dark or Light Theme

### LINKED ISSUE #213: [Bug]:  Theme Button is not at the position when open in mobile phone
### Describe the bug

- Click on the drop-down menu 
- You will see the theme button will shift from it's postion and will appear on mid left side 

### Screenshot/ Video

Screenshot (105)

### PR DESCRIPTION
**Pull request by Neeteshking21**
# Bug fixing
Fixes #213
Fixes #208
Fixes #193

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] User Interface
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I replicated the UI with the Design 

# GIF after bugs fixing and feature fixing.
issue_fix

### CODE REVIEW

**Comment by Neeteshking21** [comment]
@kaiwalyakoparkar . Please review it.

File: assets/css/style.css [hunk 1]
```diff
@@ -307,7 +308,7 @@ h6 {
 	display: flex;
 	align-items: center;
 	justify-content: space-between;
-    margin-left: 16px;
+    margin: 2px 0 0 38px;
```
Comment (kaiwalyakoparkar) [inline assets/css/style.css:311]: Please Indent the code

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Review by Neeteshking21 — COMMENTED** [review]
Indentations are perfect. It's showing because of changes
code
.

**Review by Neeteshking21 — COMMENTED** [review]
All are perfect.

**Comment by Neeteshking21** [comment]
@kaiwalyakoparkar. Please review again.🙂

**Comment by akash-kumar44** [comment]
Hey @kaiwalyakoparkar, @Neeteshking21  is sending PR's without assigning the issues to him. People who raised the issues (like #208 #213 #193 ) are asking you to assign those issues to them. On the other hand, @Neeteshking21 is sending PR's without assigning any issue to him.  This is not fair brother.

@Neeteshking21 brother please work only on those issues which are assigned to you. please don't waste the hard work of others.

**Comment by Neeteshking21** [comment]
well, I already fixed them because the rest issues are already part of #193 I found these issues later. I tried to give a message to @kaiwalyakoparkar that #213,#208 are already fixed in this PR. (No BAD INTENTION])

**Review by Neeteshking21 — COMMENTED** [review]
closing tag fixing.

**Comment by Neeteshking21** [comment]
@kaiwalyakoparkar PR is ready for review again.

**Pull request closed without merging**